t/unit/transport/test_redis.py

```diff
@@ -1311,21 +1311,21 @@ def test_mutex(self, lock_id='xxx'):
             client.setnx.return_value = True
             client.pipeline = ContextMock()
             pipe = client.pipeline.return_value
-            pipe.get.return_value = lock_id
+            pipe.get.return_value = lock_id.encode()  # redis returns bytes
```

Comment:
I guess that this happens on Python3, right? Perhaps we can use again `bytes_to_str`, so that we are always consistent.

Reply:
Yes, except that this one is `str_to_bytes`.